modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/util/PaymentProcessCommerceCheckoutStep.java

```diff
@@ -101,10 +102,8 @@ public void render(
 				(HttpServletResponse)httpServletResponseWrapper.getResponse();
 		}
 
-		String paymentServletURL =
-			originalHttpServletResponse.encodeRedirectURL(
-				paymentProcessCheckoutStepDisplayContext.
-					getPaymentServletUrl());
+		String paymentServletURL = URLCodec.encodeURL(
+			paymentProcessCheckoutStepDisplayContext.getPaymentServletUrl());
```

Comment:
hey @topolik this change seems to create problems. 
![image](https://user-images.githubusercontent.com/56033516/113016634-f824cc00-917e-11eb-9b73-60f59aa3c093.png)
Could you please check that this is the right place to encode the URL? Thx

Reply:
Yeah, I think there should be no URL encoding at this place, i.e. neither 			```originalHttpServletResponse.encodeRedirectURL``` nor ```URLCodec.encodeURL``` Thanks!

Reply:
@topolik thanks!